dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -62,4 +70,8 @@ def upload_to_s3(model_id, trained_model)
   def download_from_s3(model_id)
     AWS::S3.download_from_bucket(S3_BUCKET, model_id)
   end
+
+  def delete_from_s3(model_id)
+    AWS::S3.delete_from_bucket(S3_BUCKET, model_id)
+  end
```

Comment:
Great work! This is new to me. Would we ever need to verify/test that this action is successful in S3?

Reply:
[This delete method](https://github.com/code-dot-org/code-dot-org/blob/99238d5e702366b7cebf6a470a3080f589fa6782/lib/cdo/aws/s3.rb#L155) will return a boolean indicating whether the deletion worked or not. I took your idea and use it in the check above. Thanks!